.gitignore

```diff
@@ -3,4 +3,4 @@ node_modules/
 .vscode
 .env
 
-build/*
\ No newline at end of file
+build/*
```

Comment:
Revert changes on this file so the PR is cleaner.

Reply:
Done